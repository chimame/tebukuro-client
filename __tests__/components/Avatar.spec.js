import React from 'react'
import AvatarComponent from '../../client/components/Avatar'
import {shallow} from 'enzyme'
import toJson from 'enzyme-to-json'

describe('Avatar', () => {
  const subject = (src) => (shallow(<AvatarComponent src={src} />))

  test('render Avatar', () => {
    const src = 'example.png'
    expect(toJson(subject(src))).toMatchSnapshot()
  })

  test('not render Avatar', () => {
    const src = undefined
    expect(toJson(subject(src))).toMatchSnapshot()
  })
})
