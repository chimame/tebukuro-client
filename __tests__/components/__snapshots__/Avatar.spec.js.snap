exports[`Avatar not render Avatar 1`] = `<span />`;

exports[`Avatar render Avatar 1`] = `
<span>
  <img
    height="20"
    src="example.png"
    width="20" />
</span>
`;
